packages/tupaia-web/src/views/DashboardPDFExport.tsx

```diff
@@ -85,14 +89,15 @@ export const DashboardPDFExport = ({
 
   return (
     <Parent $isPreview={isPreview}>
-      {dashboardItems?.map(dashboardItem => (
+      {dashboardItems?.map((dashboardItem, i) => (
         <PDFExportDashboardItem
           key={dashboardItem.code}
           dashboardItem={dashboardItem}
           entityName={entity?.name}
           activeDashboard={activeDashboard}
           isPreview={isPreview}
           settings={settings}
+          displayDescription={pageIndex === 1 || (!isPreview && i === 0)}
```

Comment:
Currently (although this is due to change) there is 1 visual per page, so as long as `i===0` this should always be the same as being the first page.
Once we merge in the other ticket that I worked on with continuous pages, you can amend this.